src/Agent.Worker/Handlers/NodeHandler.cs

```diff
@@ -225,12 +225,8 @@ public string GetNodeLocation()
             string useNodeKnob = AgentKnobs.UseNode.GetValue(ExecutionContext).AsString();
 
             string nodeFolder = NodeHandler.nodeFolder;
-            if (PlatformUtil.RunningOnRHEL6 && taskHasNode16Data)
```

Comment:
>RunningOnRHEL6 [](http://example.com/codeflow?start=29&length=14)

not sure if it matters but we lost the reference to RunningOnRHEL6  in the changes

Reply:
We officially dropped RHEL6 support, so all RHEL6-related logic should be cut out completely (probably not in this PR).